### PR TITLE
Sendinblue destination

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Created doc for Sendinblue destination

### CODE REVIEW

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-device-mode.mdx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  lastName: "Keener",
+  payment: 2,
+  age: 21,
+  location: "USA",
+});
+```
+
+<div class="infoBlock">
+The Sendinblue tracker writes the <code class="inline-code">sib_cuid</code> cookie on your domain to track the user activity. As a result, you can create only one contact per session in Sendinblue using the <code class="inline-code">identify</code> call and update the same contact in that session.
+To create a new contact, you must clear the <code class="inline-code">sib_cuid</code> cookie first.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-device-mode.mdx:123]:
```suggestion
To create a new contact, you must clear the <code class="inline-code">sib_cuid</code> cookie first or use a different browser
```

Comment (Gauravudia) [reply docs/destinations/streaming-destinations/sendinblue/sendinblue-device-mode.mdx:123]: Also, As this note is specific to identify call. We can move this note under identify section

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-device-mode.mdx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+The following table details the property mappings between RudderStack and Sendinblue:
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `event` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `event` | String |
+| `context.traits`  | `properties` | Object |
+| `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name`  | `properties.FIRSTNAME` | String | 
+| `context.traits.lastName` <br/> `context.traits.lastname` <br/> `context.traits.last_name`  | `properties.LASTNAME` | String |  
+| `context.traits.phone` <br/> `context.traits.Phone` | `properties.SMS`  |  String |
+| `context.traits.email` <br/>`context.traits.Email`<br/> `context.traits.E-mail`| `properties.EMAIL`  | String |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-device-mode.mdx:87]: Please remove the email mapping from here

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-device-mode.mdx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+<div class="infoBlock">
+This section is applicable for the <code class="inline-code">identify</code> and <code class="inline-code">track</code> calls.
+</div>
+
+Sendinblue provides various [contact attributes](https://help.sendinblue.com/hc/en-us/articles/209499605-What-are-the-different-types-of-contact-attributes-) which you can use to personalize your communication with your contacts. You can find them in the [Contact Attributes](https://my.sendinblue.com/lists/add-attributes) section in the Sendinblue dashboard where `FIRSTNAME`, `LASTNAME`, and `SMS` are the default contact attributes. 
+
+You can also map the traits in RudderStack's payload to Sendinblue's contact attributes using the **Map your traits to Sendinblue contact attributes** setting in the RudderStack dashboard.
+
+- The contact attributes present in Sendinblue and sent in the `identify` call's traits are updated each time you make an `identify` call.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-device-mode.mdx:105]: You can remove this note from here

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+| `context.traits` <br/> `traits` <br/> *(excluding `firstname`, `lastname`, and `phone`)* | `attributes` | Object |
+
+
+#### Update a DOI contact
+
+The following table details the property mappings between RudderStack and Sendinblue to [update a DOI contact](https://developers.sendinblue.com/reference/updatecontact):
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> `context.externalId.id` *(when `context.externalId.type` = `contactId`)* <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `identifier`  | String |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:83]:
```suggestion
| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> `context.externalId.id` *(when `context.externalId.type` = `sendinblueContactId`)* <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `identifier`  | String |
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+| `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` | `attributes.SMS`  |  String |
+| `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name` <br/> `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name`  | `attributes.FIRSTNAME` | String |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:85]: please reverse the order. traits then contact.traits

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+#### Update a DOI contact
+
+The following table details the property mappings between RudderStack and Sendinblue to [update a DOI contact](https://developers.sendinblue.com/reference/updatecontact):
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> `context.externalId.id` *(when `context.externalId.type` = `contactId`)* <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `identifier`  | String |
+| `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` | `attributes.SMS`  |  String |
+| `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name` <br/> `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name`  | `attributes.FIRSTNAME` | String | 
+| `context.traits.lastName` <br/> `context.traits.lastname` <br/> `context.traits.last_name` <br/> `traits.lastName` <br/> `traits.lastname` <br/> `traits.last_name`  | `attributes.LASTNAME` | String |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:86]: please reverse the order. traits then contact.traits

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+
+The following table details the property mappings between RudderStack and Sendinblue to [update a DOI contact](https://developers.sendinblue.com/reference/updatecontact):
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> `context.externalId.id` *(when `context.externalId.type` = `contactId`)* <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `identifier`  | String |
+| `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` | `attributes.SMS`  |  String |
+| `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name` <br/> `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name`  | `attributes.FIRSTNAME` | String | 
+| `context.traits.lastName` <br/> `context.traits.lastname` <br/> `context.traits.last_name` <br/> `traits.lastName` <br/> `traits.lastname` <br/> `traits.last_name`  | `attributes.LASTNAME` | String | 
+| `context.traits` <br/> `traits` *(excluding `firstname`, `lastname`, and `phone`)* | `attributes` | Object |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:87]:
```suggestion
| `traits` <br/> `context.traits` | `attributes` | Object |
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+| `integrations.sendinblue.emailBlacklisted`| `emailBlacklisted` | Boolean |
+| `integrations.sendinblue.smsBlacklisted`| `smsBlacklisted` | Boolean |
+| `context.externalId.id` *(when `context.externalId.type` = `includeListIds`)* | `listIds` | Int64 array |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:90]:
```suggestion
| `context.externalId.id` *(when `context.externalId.type` = `sendinblueIncludeListId `)* | `listIds` | Int64 array |
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+| `context.externalId.id` *(when `context.externalId.type` = `unlinkListIds`)*| `unlinkListIds` | Int64 array |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:91]:
```suggestion
| `context.externalId.id` *(when `context.externalId.type` = `sendinblueUnlinkListId`)*| `unlinkListIds` | Int64 array |
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+
+#### Get contact details
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:93]: We can remove this section from here. As, we are using this api internally.

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 11]
```diff
@@ -0,0 +1,197 @@
...
+
+The following table details the property mappings between RudderStack and Sendinblue to [get contact details](https://developers.sendinblue.com/reference/getcontactinfo-1):
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` <br/> `context.externalId.id` *(when `context.externalId.type` = `contactId`)* <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `identifier`  | String |
+
+#### Delete contact from list
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:101]: Please add a note ->
`Rudder will use this API and delete a contact from a given lists if DOI is disabled in webapp and sendinblueUnlinkListId is provided through externalId`

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 12]
```diff
@@ -0,0 +1,197 @@
...
+
+The following table details the property mappings between RudderStack and Sendinblue to [delete a contact from list](https://developers.sendinblue.com/reference/removecontactfromlist):
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` <br/>  <span style="color: #4D4DFF;font-size:12px;">Required, if contactID is not present.</span>| `emails`  | String |
+| `context.externalId.id` *(when `context.externalId.type` = `contactId`)* | `ids`  | String |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:108]:
```suggestion
| `context.externalId.id` *(when `context.externalId.type` = `sendinblueContactId`)* | `ids`  | String |
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 13]
```diff
@@ -0,0 +1,197 @@
...
+
+
+#### Update a DOI contact
+
+The following table details the property mappings between RudderStack and Sendinblue to [update a DOI contact](https://developers.sendinblue.com/reference/updatecontact):
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> `context.externalId.id` *(when `context.externalId.type` = `contactId`)* <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `identifier`  | String |
+| `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` | `attributes.SMS`  |  String |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:84]:
```suggestion
| `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` | `attributes.SMS`  |  String |
| `traits.newEmail` <br/> `context.traits.newEmail` | `attributes.EMAIL`  | String |
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 14]
```diff
@@ -0,0 +1,197 @@
...
+    location: "USA",
+  }, {
+    externalId: [{
+        type: "includeListIds",
+        id: 1,
+      },
+      {
+        type: "includeListIds",
+        id: 2,
+      },
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:36]:
```suggestion
    externalId: [{
        type: "sendinblueIncludeListIds",
        id: [1,2],
      },
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 15]
```diff
@@ -0,0 +1,197 @@
...
+
+
+A sample `identify` call is shown below:
+
+```javascript
+rudderanalytics.identify(
+  "alex@example.com", {
+    firstName: "Alex",
+    lastName: "Keener",
+    phone: "+1-202-555-0146",
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:24]:
```suggestion
    phone: "+12025550146",
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 16]
```diff
@@ -0,0 +1,197 @@
...
+#### Create contact
+
+The following table details the property mappings between RudderStack and Sendinblue to [create a contact](https://developers.sendinblue.com/reference/createcontact):
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> <span style="color: #4D4DFF;font-size:12px;">Required, if phone number is not present.</span>| `email`  | String |
+| `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` <br/>  <span style="color: #4D4DFF;font-size:12px;">Required, if email is not present.</span>| `attributes.SMS`  |  String |
+| `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name` <br/> `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name`  | `attributes.FIRSTNAME` | String | 
+| `context.traits.lastName` <br/> `context.traits.lastname` <br/> `context.traits.last_name` <br/> `traits.lastName` <br/> `traits.lastname` <br/> `traits.last_name`  | `attributes.LASTNAME` | String |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:55]:
```suggestion
| `traits.lastname` <br/> `traits.last_name` <br />`context.traits.lastName` <br/> `context.traits.lastname` <br/> `context.traits.last_name` <br/> `traits.lastName`   | `attributes.LASTNAME` | String | 
```

[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:55]: Done.

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 17]
```diff
@@ -0,0 +1,197 @@
...
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `email`  | String |
+| `context.externalId.id` *(when `context.externalId.type` = `includeListIds`)* <span style="color: #4D4DFF;font-size:12px;">Required</span> | `includeListIds` | Int64 Array (ID of the list to add the contact) |
+| `context.externalId.id` *(when `context.externalId.type` = `templateId`)* <br/> OR <br/> `templatedId` *(defined in RudderStack dashboard)* <span style="color: #4D4DFF;font-size:12px;">Required</span> | `templateId` | Int64 (DOI template's ID) |
+| Redirection URL *(defined in RudderStack dashboard)* <span style="color: #4D4DFF;font-size:12px;">Required</span>| `redirectionUrl` | URL |
+| `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` | `attributes.SMS`  |  String |
+| `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name` <br/> `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name`  | `attributes.FIRSTNAME` | String |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:72]:
```suggestion
| `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name` <br /> `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name`   | `attributes.FIRSTNAME` | String | 
```

[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:72]: Done.

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 18]
```diff
@@ -0,0 +1,197 @@
...
+| `context.externalId.id` *(when `context.externalId.type` = `includeListIds`)* <span style="color: #4D4DFF;font-size:12px;">Required</span> | `includeListIds` | Int64 Array (ID of the list to add the contact) |
+| `context.externalId.id` *(when `context.externalId.type` = `templateId`)* <br/> OR <br/> `templatedId` *(defined in RudderStack dashboard)* <span style="color: #4D4DFF;font-size:12px;">Required</span> | `templateId` | Int64 (DOI template's ID) |
+| Redirection URL *(defined in RudderStack dashboard)* <span style="color: #4D4DFF;font-size:12px;">Required</span>| `redirectionUrl` | URL |
+| `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` | `attributes.SMS`  |  String |
+| `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name` <br/> `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name`  | `attributes.FIRSTNAME` | String | 
+| `context.traits.lastName` <br/> `context.traits.lastname` <br/> `context.traits.last_name` <br/> `traits.lastName` <br/> `traits.lastname` <br/> `traits.last_name`  | `attributes.LASTNAME` | String |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:73]:
```suggestion
| `traits.lastName` <br/> `traits.lastname` <br/> `traits.last_name` <br/>  `context.traits.lastName` <br/> `context.traits.lastname` <br/> `context.traits.last_name`   | `attributes.LASTNAME` | String | 
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx (hunk 5 above)
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:85]:
```suggestion
| `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name` <br/> `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name`    | `attributes.FIRSTNAME` | String | 
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx (hunk 6 above)
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:86]:
```suggestion
| `traits.lastName` <br/> `traits.lastname` <br/> `traits.last_name` <br/> `context.traits.lastName` <br/> `context.traits.lastname` <br/> `context.traits.last_name`    | `attributes.LASTNAME` | String | 
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx (hunk 10 above)
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:93]:
```suggestion
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 19]
```diff
@@ -0,0 +1,197 @@
...
+| `context.traits.lastName` <br/> `context.traits.lastname` <br/> `context.traits.last_name` <br/> `traits.lastName` <br/> `traits.lastname` <br/> `traits.last_name`  | `attributes.LASTNAME` | String | 
+| `context.traits` <br/> `traits` *(excluding `firstname`, `lastname`, and `phone`)* | `attributes` | Object |
+| `integrations.sendinblue.emailBlacklisted`| `emailBlacklisted` | Boolean |
+| `integrations.sendinblue.smsBlacklisted`| `smsBlacklisted` | Boolean |
+| `context.externalId.id` *(when `context.externalId.type` = `includeListIds`)* | `listIds` | Int64 array |
+| `context.externalId.id` *(when `context.externalId.type` = `unlinkListIds`)*| `unlinkListIds` | Int64 array |
+
+#### Get contact details
+
+The following table details the property mappings between RudderStack and Sendinblue to [get contact details](https://developers.sendinblue.com/reference/getcontactinfo-1):
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:95]:
```suggestion
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 20]
```diff
@@ -0,0 +1,197 @@
...
+
+| RudderStack property | Sendinblue property | Data type |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:97]:
```suggestion
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 21]
```diff
@@ -0,0 +1,197 @@
...
+| :-----| :--------| :------|
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:98]:
```suggestion
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 22]
```diff
@@ -0,0 +1,197 @@
...
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` <br/> `context.externalId.id` *(when `context.externalId.type` = `contactId`)* <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `identifier`  | String |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:99]:
```suggestion
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx (hunk 10 above)
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:93]: Okay, sure.

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 23]
```diff
@@ -0,0 +1,197 @@
...
+
+#### Delete contact from list
+
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:102]:
```suggestion

<div class="warningBlock">
RudderStack uses this API to delete a contact from a given list <strong>if</strong> the <strong>Create contact via Double-opt-in</strong> setting is disabled in the dashboard and <code class="inline-code">sendinblueUnlinkListIds</code> is provided in the event's <code class="inline-code">externalId</code> field.
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 24]
```diff
@@ -0,0 +1,197 @@
...
+### Supported mappings
+
+The following table details the property mappings between RudderStack and Sendinblue:
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `context.traits.email` <br/> `properties.email`<br/> `context.traits.phone` <br/> `properties.phone` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `email` | String |
+| `properties.title` <br/> `context.page.title`| `properties.ma_title`  | String |
+| `properties.url` <br/> `context.page.url` | `page` | String |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:193]:
```suggestion
| `properties.url` <br/> `context.page.url` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `page` | String | 
| `properties.title` <br/> `context.page.title`| `properties.ma_title` | String |
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 25]
```diff
@@ -0,0 +1,197 @@
...
+```
+
+### Supported mappings
+
+The following table details the property mappings between RudderStack and Sendinblue:
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `context.traits.email` <br/> `properties.email`<br/> `context.traits.phone` <br/> `properties.phone` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `email` | String |
+| `properties.title` <br/> `context.page.title`| `properties.ma_title`  | String |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:192]:
```suggestion
```

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx (hunk 24 above)
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:193]: Sure - updated accordingly.

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx [hunk 26]
```diff
@@ -0,0 +1,194 @@
...
+#### Create or update a contact
+
+The following table details the property mappings between RudderStack and Sendinblue to [create or update a contact](https://developers.sendinblue.com/reference/createcontact):
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `traits.email` <br/> `context.traits.email` <br/> `properties.email` <br/> <span style="color: #4D4DFF;font-size:12px;">Required, if phone number is not present.</span>| `email`  | String |
+| `traits.phone` <br/> `context.traits.phone` <br/> `properties.phone` <br/>  <span style="color: #4D4DFF;font-size:12px;">Required, if email is not present.</span>| `attributes.SMS`  |  String |
+| `traits.newEmail` <br/> `context.traits.newEmail` | `attributes.EMAIL`  | String |
+| `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name` <br/> `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name`  | `attributes.FIRSTNAME` | String |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-cloud-mode.mdx:51]:
```suggestion
| `traits.firstName` <br/> `traits.firstname` <br/> `traits.first_name` <br/>  `context.traits.firstName` <br/> `context.traits.firstname` <br/> `context.traits.first_name`   | `attributes.FIRSTNAME` | String | 
```

**Comment by ameypv-rudder** [comment]
@Gauravudia all your suggestions and changes have been incorporated. As discussed, closing this PR - we will port the changes in a fresh PR once the new docs site is up and running. cc @ishachopra0401

File: docs/destinations/streaming-destinations/sendinblue/sendinblue-device-mode.mdx [hunk 27]
```diff
@@ -0,0 +1,145 @@
...
+rudderanalytics.page("Home")
+```
+
+### Supported mappings
+
+The following table details the property mappings between RudderStack and Sendinblue:
+
+| RudderStack property | Sendinblue property | Data type |
+| :-----| :--------| :------|
+| `name` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `page_name` | String |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/sendinblue/sendinblue-device-mode.mdx:140]: name is an optional field in page call

**Review by Gauravudia — APPROVED** [review]

**Pull request merged**